controllers/helpers.js

```diff
@@ -34,8 +34,10 @@ const getPaginationElements = query => {
     query.limit && !Number.isNaN(parseInt(query.limit, 10)) ? parseInt(query.limit, 10) : Infinity;
   const limit = Math.min(queryLimit, 1000);
   const offset = (query.offset && parseInt(query.offset, 10)) || 0;
+  const sortBy = query.sortBy || 'id';
```

Comment:
Could we make it so it always sorts by `createdAt DESC` as a default, and as a secondary sort when a different one is specified?

So if you specify no sort, the sort will be

 `ORDER BY "Cases".createdAt DESC`
 
 But if you specify to sort by status, the order will be
 
 `ORDER BY "Cases".status ASC, "Cases".createdAt DESC`
 
 So all the open cases aren't jumbled up in a random order etc
 
 On second thoughts maybe it's better to stick with `ìd` rather than `createdAt` for the time being, since `createdAt` is not indexed. I want to move to `createdAt` in case we move away from an ascending integer as an ID, but that can wait until we review the indexes.

Reply:
I made `sortBy` defaults to `id` because that was specified in the Jira ticket. Right now, the result is the same as using `createdAt`, because the ids are sequential. But, if we change ids to not be sequential anymore, than it may get a little bit confusing.

Reply:
Yes Id is fine for now, it's keeping it as a secondary sort clause when you are sorting on something else is the important thing